libs/common/src/sql-db-entities/partial-entities/month-and-year.partial-entity.ts

```diff
@@ -1,9 +1,15 @@
 import { Column } from 'typeorm';
 
 export class MonthAndYearPartialEntity {
+  /**
```

Comment:
Minor touches to the type.

```typescript
/**
   * The year
   * @required true
   * @example 2024
   */
```

Is the month stored in JS month number (0 - Jan, 11 - Dec) or standard (1-12), an explanation on that will be great. Thanks

Reply:
As far as I can see, `@required` is not a JSDoc tag. I will make the change documenting it as a one-indexed month though

Reply:
Much cleaner, thanks.